### ISSUE TITLE
timestampsInSnapshots - Cloud Firestore Settings

### ISSUE THREAD

**Issue by ggabrovski** [firebase/firebase-dart#171]
Custom configurations in Cloud Firestore Settings class for `timestampsInSnapshots`.
Update interop/firestore_interop.dart in relation with
https://firebase.google.com/docs/reference/js/firebase.firestore.Settings
See also: https://github.com/firebase/firebase-js-sdk/blob/master/packages/firestore/CHANGELOG.md#040

**Comment by kevmoo** [comment]
Pull requests welcome!

**Comment by kevmoo** [comment]
Yup. See that here - https://firebase.google.com/support/release-notes/js#4.13.0

**Comment by kevmoo** [comment]
This will be fixed in https://github.com/firebase/firebase-dart/pull/181
